src/ui/i18n/translationflagger.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * A placeholder for Translation calls handled during preprocessing.
+ *
+ * Any calls of this method will be removed during a preprocessing step during SDK
+ * bundling. To support cases where someone may want to bundle without using our
+ * bundling tasks, this functions simply returns the phrase passed in.
+ *
+ * @param {string} phrase
+ * @param {string} pluralPhrase
+ * @param {string | number} count
+ * @param {string} context
+ * @param {Object} interpolationValues
+ * @returns {string}
+ */
+export function replaceWithTranslation ({ phrase, pluralPhrase, count, context, interpolationValues }) {
+  return phrase;
```

Comment:
As Bryan noted, we want to handle things gracefully if these calls don't get transpiled out. Obviously, we won't give them a translated version of their phrase, but I think we could still pluralize/interpolate here. That would allow us to handle the no-transpilation case even more gracefully.